src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -380,26 +383,18 @@ func (cfg Config) fetchCloudConfigForAddr(proxyAddr string) ([]byte, error) {
 	if err != nil {
 		return nil, fmt.Errorf("Unable to initialize HTTP client: %s", err)
 	}
-	// Try and wait to get geolocated, up to a point.  Waiting indefinitely
-	// might prevent us from ever getting geolocated if our current
-	// configuration is preventing domain fronting from working.
```

Comment:
I think this comment had a valid point.  If we wait indefinitely for geolookup, but the (domain fronted) geolookup server can't be reached for any reason related to the current config, then the client can never recover from that.